code/modules/antagonists/heretic/knowledge/cosmic_lore.dm

```diff
@@ -161,7 +169,7 @@
 	var/mob/living/third_target_resolved = third_target?.resolve()
 	var/need_mob_update = FALSE
 	need_mob_update += target.adjustFireLoss(4, updating_health = FALSE)
-	need_mob_update += target.adjustCloneLoss(2, updating_health = FALSE)
+	need_mob_update += target.adjustOrganLoss(pick(valid_organ_slots), 8)
 	if(need_mob_update)
 		target.updatehealth()
 	if(target == second_target_resolved || target == third_target_resolved)
```

Comment:
If the second or third target are different mobs, then why are we applying damage to them and then updating the original targets health?

Reply:
Seems to be oversights from the previous refactor, I'll fix them tomorrow while I'm already at it